csv_export_invoice/models/csv_export_invoice.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import _, models
+from openerp.exceptions import ValidationError
+
+HEADERS = (
+    "journal",
+    "reference",
+    "date_invoice",
+    "date_due",
+    "partner_reference",
+    "structured_communication",
+    "comment",
+    "total_amount",
+    "account",
+    "base_amount",
+    "vat_code",
+    "vat_amount",
+    "line_comment",
+    "ana1",
+    "ana2",
+    "ana3",
+    "ana4",
+    "ana5",
+    "ana6",
+)
+
+
+class InvoiceCSVExport(models.TransientModel):
+    _name = "csv.export.invoice"
+    _inherit = "csv.export.base"
+    _description = "Export Invoice CSV"
+    _connector_model = "account.invoice"
+    _filename_template = "INV_%Y%m%d_%H%M.csv"
+
+    def get_domain(self):
+        return [
+            ("state", "!=", "draft"),
+            ("state", "!=", "cancel"),
+            ("date", ">=", self.start_date),
+            ("date", "<", self.end_date),
+        ]
+
+    def get_headers(self):
+        return HEADERS
+
+    def get_row(self, record):
+        line = record
+        invoice = line.invoice_id
+
+        if invoice.reference_type == "bba":
+            reference = invoice.reference
+        else:
+            reference = ""
+
+        tax = line.invoice_line_tax_ids
+        if len(tax) > 1:
+            raise ValidationError(
+                _("This module does not allow several taxes per invoice line")
+            )
+        elif tax:
+            tax_code = tax.name  # todo map code
+            tax_amount = line.price_subtotal * (21.0 / 100)
```

Comment:
I'm a bit disturbing seeing 21% tax hard coded... what if the tax is 6 or 12% or any other tax

Reply:
Good catch, it's fixed :-)